src/main/java/com/hedvig/contract/shared/switchers/SwitcherEmailServiceImpl.kt

```diff
@@ -50,16 +51,38 @@ class SwitcherEmailServiceImpl(
   @Transactional
   override fun sendEmail(email: SwitcherEmail) {
     logger.info("Attempting to send cancellation email ${email.id}")
-    val contents = mailer.sendEmail(
-      switcherCompany = email.switcherCompany!!,
-      memberId = email.memberId,
-      ssn = memberService.getMember(email.memberId).ssn ?: throw NoSsnPresentException("No SSN was present for member ${email.memberId}"),
-      insuranceMandatePdfContent = contractRepository
-        .findByMemberId(email.memberId)
-        .orElseThrow { NoInsuranceMandatePresentException("No insurance mandate present for member ${email.memberId}") }
-        .contract
-    )
-    logger.info("Sent cancellation email ${email.id}")
+
+    val member = memberService.getMember(email.memberId)
+
+    val contents: String = when (email.lineOfBusiness) {
+      SwitcherLineOfBusiness.NORWEGIAN_HOME_CONTENT,
+      SwitcherLineOfBusiness.NORWEGIAN_TRAVEL -> {
+
+        val norwegianEmailData = NorwegianEmailData.from(member, email)
+        val content = mailer.sendNorwegianEmail(norwegianEmailData)
+        logger.info("Sent cancellation email ${email.id}")
+        content
+      }
+      SwitcherLineOfBusiness.SWEDISH_APARTMENT,
+      SwitcherLineOfBusiness.SWEDISH_HOUSE -> {
+
+        val insuranceMandate = contractRepository.findByMemberId(email.memberId)
+          .orElseThrow { NoInsuranceMandatePresentException("No insurance mandate present for member" +
+            " ${email.memberId}")
+          }.contract
+
+        val swedishEmailData = SwedishEmailData(
+          switcherCompany = email.switcherCompany!!,
+          memberId = email.memberId,
+          ssn = member.ssn ?: throw NoSsnPresentException("No SSN was present for member ${email.memberId}"),
+          insuranceMandatePdfContent = insuranceMandate
```

Comment:
```suggestion
          contract = insuranceMandate
```
This is also a contract?